packages/suite/src/components/suite/StakeAmountWrapper.tsx

```diff
@@ -0,0 +1,40 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+import { Tooltip, variables } from '@trezor/components';
+import { Translation } from './Translation';
+
+const Container = styled.div`
+    position: relative;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    margin: -2px -12px;
+    padding: 2px 12px;
+    border-radius: 6px;
+    transition: background 0.1s ease-in;
+    cursor: pointer;
```

Comment:
nit: I would take user to staking tab on click

Reply:
https://github.com/everstake/trezor-suite/pull/5/commits/25028a4faf3c04394eda3e5a7fee23435adbf3ef